server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -802,41 +175,24 @@ export function configureWebSocketServices(
 				),
 				[CommonProperties.roomClients]: JSON.stringify(Array.from(roomMap.keys())),
 				[BaseTelemetryProperties.correlationId]: correlationId,
-			});
+			};
 
 			connectDocumentP = getGlobalTelemetryContext().bindPropertiesAsync(
 				{ correlationId, ...baseLumberjackProperties },
 				async () =>
-					connectDocument(connectionMessage)
+					connectDocument(
+						socket,
+						lambdaSettings,
+						lambdaConnection,
+						connectionMessage,
+						properties,
+					)
```

Comment:
IMO these are fine because each is a strongly typed object, except for the lumber properties, so it is clear which param is which. I don't want to change all of the params of all the functions to be named from positional in this PR.